client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -123,6 +108,22 @@ const wrapSettingsForm = ( getFormSettings ) => ( SettingsForm ) => {
 			}
 		}
 
+		displaySuccessNotice( noticeSettings ) {
+			const launchpadTasksToComplete = [];
+
+			Object.entries( FIELDS_TO_LAUNCHPAD_TASKS ).forEach( ( [ field, taskSlug ] ) => {
+				if ( get( this.state.modifiedFields, field ) ) {
```

Comment:
We're trying to get away from using lodash for things like this p4TIVU-9Bf-p2

Reply:
Even though this file makes extensive use of lodash :) May as well not add to it

Reply:
Ok, scratch that 😅 I see what we're using `get()` now: the dotted `"subscription_options.welcome"` field name.

I guess that's why p4TIVU-9Bf-p2 said we might need to add `get()` to `@automattic/js-utils`.